WordPress/src/main/res/values/dimens.xml

```diff
@@ -81,11 +81,11 @@
     <dimen name="margin_large">12dp</dimen>
     <dimen name="margin_extra_large">16dp</dimen>
     <dimen name="margin_extra_medium_large">24dp</dimen>
-    <dimen name="margin_extra_extra_medium_large">32dp</dimen>
```

Comment:
This dimension might have been removed accidentally.  It looks like it's still used [here](https://github.com/wordpress-mobile/WordPress-Android/pull/10362/files?file-filters%5B%5D=.java&file-filters%5B%5D=.xml#diff-7c2f033014a455e52db8dd994c5f3e58R35) and [here](https://github.com/wordpress-mobile/WordPress-Android/pull/10362/files?file-filters%5B%5D=.java&file-filters%5B%5D=.xml#diff-7c2f033014a455e52db8dd994c5f3e58R116).  The same dimension is defined in `WordPressLoginFlow` so those instances are using that reference.  Should we add it back to `WordPress` so that it is explicitly defined in the main module too?

Reply:
Not sure how it even got removed. Mystery.